index.html

```diff
@@ -12,39 +12,126 @@
+                <img src="images/github.png" class="social" alt="Github"></a>
+            </li>
+            <li>
+              <a href="https://www.linkedin.com/" target="_blank"  rel="noopener noreferrer">
+                <img src="images/Linkedin.png" class="social" alt="Linkedin"></a>
+            </li>                               
+            <li>
+              <a href="https://www.angellist.com/" target="_blank"  rel="noopener noreferrer">
+                <img src="images/angellist.png" class="social" alt="Angellist"></a>
+            </li>
+            <li>
+              <a href="https://www.twitter.com/" target="_blank"  rel="noopener noreferrer">
+                <img src="images/twitter.png"  class="social" alt="Twitter"></a>
+            </li>
+            <li>
+              <a href="https://www.medium.com/" target="_blank"  rel="noopener noreferrer">
+                <img src="images/medium.png" class="social" alt="Medium"></a>
+            </li>
+          </ul>
+        </div>
+      </div> 
+    </section>
+  
+    <section class="work-section" id="work-section">        
+    </section>
+    
+    <section id="popup-work-section"> 
+      <div class="pop-title">
+        <h2>Tonic</h2>
+        <a href="javascript:void(0);" id="pop-close" onclick="WorkCards()">
+          <img  src="./images/pop-close.svg"  alt="close">
+        </a>
+        
+      </div>
+      <div class="pop-infoline">
+        <h3>CANOPY</h3>
+        <img src="images/dot.png" alt="">
+        <h4>Back End Dev</h4>
+        <img src="images/dot.png" alt="">
+        <h4>2015</h4>
+      </div>
+
+      <div class="pop-img" >
+        <img src="images/work1.svg" alt="Work 1">
+      </div>    
+
+      <div class="pop-details">
+        <div class="pop-text">
+          <p>Lorem Ipsum is simply dummy text of 
+            the printing and typesetting industry. Lorem Ipsum has 
+            been the industry's standard dummy text ever since the 
+            1500s, when an unknown printer took a galley of type 
+            and scrambled it to make a type specimen book. It has
+            survived not only five centuries,but also the leap 
+            into electronic typesetting remaining essent
           </p>
-          <h3>LET'S CONNECT</h3>
-          <div>
-            <ul class="icons-social" >
+        </div> 
+        <div class="pop-languages">                
+          <ul>
+            <li><img src="images/html-tag.png" alt="html"></li>
+            <li><img src="images/css-tag.png" alt="css"></li>
+            <li><img src="images/js-tag.png" alt="javaScript"></li>
+            <li></li>
+          </ul>
+          <div class="pop-buttons">
+            <button type="button" class="pop-btn">See Live<img src="images/popup-btn-Icon.png" alt=""></button>
+            <button type="button" class="pop-btn">See Source<img src="images/popup-github.png" alt=""></button>
+          </div>
+        </div>
+      </div>
+    </section> 
```

Comment:
- [ ] Kindly be noticed that the modal popup data is populated with the data from the projects array.
- [ ] So when the user clicks on a project it should load this specific project.